pkgs/development/libraries/physics/apfel/default.nix

```diff
@@ -11,13 +11,15 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-fRdJ+C92tEC75iUwP9Tmm/EswrlA52eUo5fBjfieH9o=";
   };
 
+  nativeBuildInputs = [ autoreconfHook ];
```

Comment:
Can we make this optional, or add a comment that this is for aarch64-darwin

Reply:
Done.